### PR TITLE
9004: AC Tests

### PR DESCRIPTION
**Pull request by RuthBerry96**
Story: [AB#9004](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/9004)
Task: [AB#9319](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/9319)

### CODE REVIEW

File: src/OrderFormAcceptanceTests.TestData/SupplierInfo.cs [hunk 1]
```diff
@@ -70,5 +70,12 @@ private static IEnumerable<SupplierDetails> SupplierLookup(string connectionStri
 
             return SqlExecutor.Execute<SupplierDetails>(connectionString, query, new { catalogueItemType = (int)catalogueItemType, provisioningType = (int)provisioningType });
         }
+
+        public static int SupplierWithMoreThanOneSolution(string connectionString)
+        {
+            const string query = @"SELECT SupplierId, COUNT(*) FROM CatalogueItem WHERE CatalogueItemTypeId = 1 GROUP BY SupplierId ORDER BY 2 DESC";
```
Comment (talawson05) [inline src/OrderFormAcceptanceTests.TestData/SupplierInfo.cs:76]: Didn't Sam create some enums somewhere which are more descriptive for the CatalogueItemTypeId

Comment (RuthBerry96) [reply src/OrderFormAcceptanceTests.TestData/SupplierInfo.cs:76]: Yes, just found them! Will change it to include that. Thanks 😄

**Review by talawson05 — APPROVED** [review]
LGTM

**Review by talawson05 — CHANGES_REQUESTED** [review]
conflicts, sorry about that. I've merged in some other branches

**Comment by RuthBerry96** [comment]
> conflicts, sorry about that. I've merged in some other branches

I've just merged. It sound be fine now I think

**Review by talawson05 — APPROVED** [review]

**Pull request merged**